lib/elastic_apm/process_info.rb

```diff
@@ -8,9 +8,11 @@ def initialize(config)
     end
 
     def build
+      pid = $PID || Process.pid
```

Comment:
I guess `Process.pid` is guaranteed to always have a value?

I'm just wondering why we even check `$PID` first

Reply:
Because `Process.pid` is [not available on all platforms](http://ruby-doc.org/core-2.2.0/Process.html#method-c-pid).

Reply:
ah, gotcha! LGTM then